### ISSUE TITLE
Duration of each test when using SerenityParameterizedRunner is always 0

### ISSUE THREAD

**Issue by dsmt** [serenity-bdd/serenity-core#180]
In report of parameterized tests the duration is always zero, while steps have their duration.

![](http://take.ms/RMgn5)

![](http://take.ms/dzzXE)

**Comment by YamStranger** [comment]
Hello @dsmt!

Thanks for reporting us. 

Fix for this issue included in 1.1.22-rc.5 release. It should work. 

To check it you can use our demo project [junit-parameterized-tests](https://github.com/serenity-bdd/serenity-demos/tree/master/junit-parameterized-tests) with 1.1.22-rc.5 release. 

For this class 
```
@RunWith(SerenityParameterizedRunner.class)
@UserStoryCode("US01")
@Concurrent
public class SampleTest {
    @TestData
    public static Collection<Object[]> testData() {
        return Arrays.asList(new Object[][]{{1}, {2}, {3}});
    }

    @Steps
    TestSteps steps;
    private final int value;

    public SampleTest(int value) {
        this.value = value;
    }

    @Test
    public void shouldExecuteThisTest() {
        steps.initialization();
        steps.when_example_action_for(this.value);
        steps.then_example_result_should_be(this.value);
    }

    @Test
    public void shouldExecuteThisTestToo() {
        steps.initialization();
        steps.when_example_action_for(this.value);
        steps.then_example_result_should_be(this.value);
    }
}
```

our reports will look like for this test:
![report_1](https://cloud.githubusercontent.com/assets/1667699/11265093/09f14264-8ea2-11e5-91df-79ae79d467c5.png)
![report_2](https://cloud.githubusercontent.com/assets/1667699/11265092/09efd3ca-8ea2-11e5-9936-6dbe0f0398c7.png)

**Comment by dsmt** [comment]
Great work! Thanks!
